src/main/java/redis/clients/jedis/JedisPool.java

```diff
@@ -26,29 +26,41 @@ public JedisPool(String host, int port) {
     this(new GenericObjectPoolConfig(), host, port);
   }
 
+  /**
+   * @param host
+   * @deprecated This constructor will not accept a host string in future. It will accept only a uri
+   * string. You can use {@link JedisURIHelper#isValid(java.net.URI)} before this.
+   */
+  @Deprecated
   public JedisPool(final String host) {
```

Comment:
@sazzad16 you should rename the attribute from `host` to `uri` if that is the future